R/zzz.R

```diff
@@ -18,17 +18,17 @@ gradethis_default_options <- list(
 
 
   ### legacy ###
-  gradethis_glue_correct = "{ random_praise() } { .message } { .correct }",
-  gradethis_glue_incorrect = "{ .message } { .incorrect } { random_encouragement() }",
+  gradethis_glue_correct = "{random_praise()} {.message} {.correct}",
+  gradethis_glue_incorrect = "{pipe_warning()}{.message} {.incorrect} {random_encouragement()}",
 
-  gradethis_glue_pipe = paste0(
```

Comment:
Should we put in the news that this is removed?

Reply:
Good idea, thanks!